src/core_plugins/kibana/public/visualize/listing/visualize_landing_page_table.js

```diff
@@ -170,18 +212,26 @@ export class VisualizeLandingPageTable extends React.Component {
       <LandingPageToolBar
         filter={filter}
         onFilter={this.onFilter}
-        pagerState={pager}
+        startNumber={0}
+        endNumber={0}
+        totalItems={10}
+        hasPreviousPage={this.hasPreviousPage}
+        hasNextPage={this.hasNextPage}
         onNextPage={this.turnToNextPage}
         onPreviousPage={this.turnToPreviousPage}
         actionButtons={this.getActionButtons()}/>
       {
         this.getTableContents()
       }
       <LandingPageToolBarFooter
-        pagerState={pager}
+        startNumber={0}
+        endNumber={0}
+        totalItems={10}
+        hasPreviousPage={this.hasPreviousPage}
+        hasNextPage={this.hasNextPage}
```

Comment:
The duplication of the props here and for `LandingPageToolBar` make me question this abstraction. Maybe we should use composition instead and create an abstraction called `getPagerButtons()` which will render the pagination buttons and provide them to  `LandingPageToolBar` and  `LandingPageToolBarFooter` as children, or as a prop?